parse/table_definitions_avalanche/pangolin/UniswapV2Pair_event_Approval.json

```diff
@@ -0,0 +1,54 @@
+{
+    "parser": {
+        "type": "log",
+        "contract_address": "select pair from ref('uniswapv2factory_event_paircreated')",
```

Comment:
Not sure if it's ok for this to be all lowercase.

Reply:
SQL can be lowercase, but the table name should match the case i.e. UniswapV2Factory_event_PairCreated

Reply:
Sounds good, I'll update by hand and will look at updating the table generation tool.

Reply:
Updated in ccd9680.